package.json

```diff
@@ -22,6 +22,7 @@
     "tachyons-sass": "^4.9.3"
   },
   "devDependencies": {
+    "tachyons-typography": "^3.0.5",
```

Comment:
It looks like `tachyons-typography` is already part of `tachyons-sass`, so we don't need to specify it as another dependency! `truncate` will work without this line too :)